api/CONFIG.md

```diff
@@ -62,6 +62,14 @@ root | "/" |  Make root open index.html
 default_index | "" |  Set default 'index.html' path to open for implicit routes
 watch | false |  Enable watch mode
 
+# `webview.navigator.mounts`
```

Comment:
@chicoxyzzy this looks weird - maybe I should just omit the sample values...

Reply:
what's weird? it looks good to me